tests/TestSensor.py

```diff
@@ -6,10 +6,9 @@ class TestSensor(Doberman.LANSensor):
     def set_parameters(self):
         self._msg_start = '*'
         self._msg_end = '\r\n'
-        self.reading_commands = {'one' : 'READ:one',
-                                 'two' : 'READ:two'}
-        self.reading_pattern = re.compile(('OK;(?P<value>%s)' %
-            Doberman.utils.number_regex).encode())
+        self.reading_commands = {'t_in' : 'READ:t_in',
+                                 't_amb' : 'READ:t_amb',
+                                 'q_add': 'READ:q_add'}
+        self.reading_pattern = re.compile(f'OK;(?P<value>{Doberman.utils.number_regex})'.encode())
         self.command_patterns = [
-                (re.compile('d(?P<sides>[1-9][0-9]*)'),
-                    lambda m : f'SET:sides={m.group("sides")}')]
+            (re.compile(f'SET q_add=({Doberman.utils.number_regex})', lambda m : f'SET:q_add={m.group(1)}'))]
```

Comment:
The parantheses are wrong, I think. Shouldnt it be:
` (re.compile(f'SET q_add=({Doberman.utils.number_regex})'), lambda m : f'SET:q_add={m.group(1)}')]`

Reply:
Ideally yes but I'll make it compatible with #117